catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/impl/ResourceCacheImplTest.java

```diff
@@ -409,17 +426,29 @@ public void containsSpecificResourceNotInCache() {
     }
 
     private Metacard createMetacard(String sourceId, String metacardId) throws URISyntaxException {
-        
+
         Metacard metacard = generateMetacard();
         metacard.setSourceId(sourceId);
         metacard.setAttribute(new AttributeImpl(Metacard.ID, metacardId));
         return metacard;
     }
 
     private ReliableResource createCachedResource(Metacard metacard) {
-        String fileName = "15bytes.txt";
+        String fileName = "10bytes.txt";
```

Comment:
Had to redefine the resource used. On Windows, test resources were "disappearing" with every run of the test class.